dmutils/__init__.py

```diff
@@ -4,4 +4,4 @@
 import flask_featureflags  # noqa
 
 
-__version__ = '36.0.1'
+__version__ = '37.0.1'
```

Comment:
Major bump? Is this a breaking change?

Reply:
Ooh, I think I thought I was going to make it one and then decided not to. It can just be a minor, I think.